pulsar-functions/utils/src/test/java/org/apache/pulsar/functions/utils/functioncache/FunctionClassLoadersTest.java

```diff
@@ -34,23 +35,24 @@
 
     @Test
     public void testCreateClassLoader() throws Exception {
-        URL jarUrl = getClass().getClassLoader().getResource("multifunction.jar");
+        URL jarUrl = FileSystems.getDefault()
```

Comment:
with this change, the tests are not accurate anymore. at the moment you are adding source files or including it as a dependency, those classes are already loaded by the class loader. That says `AddFunction` is loaded by `getClass().getClassLoader()`. this doesn't really test the user-defined jar anymore.

so I would suggest just remove multifunctions.jar and its related test cases.

Reply:
Fine for me